vowpalwabbit/recall_tree.cc

```diff
@@ -313,7 +312,7 @@ predict_type predict_from(recall_tree& b, single_learner& base, example& ec, uin
   MULTICLASS::label_t mc = ec.l.multi;
   uint32_t save_pred = ec.pred.multiclass;
 
-  ec.l.simple = {FLT_MAX, 0.f, 0.f};
+  ec.l.simple = {FLT_MAX, VW::UNUSED_1, VW::UNUSED_0};
```

Comment:
`UNUSED_0` for both?

Reply:
oh, they are unused members so I guess it doesn't matter?

Reply:
Addressed this in an earlier comment.  This is deliberate.